tcms/rpc/api/testcase.py

```diff
@@ -49,6 +49,7 @@ def add_component(case_id, component):
         :param component: Name of Component to add
         :type component: str
         :return: Serialized :class:`tcms.testcases.models.TestCase` object
+        :rtype: dict
         :raises: PermissionDenied if missing the *testcases.add_testcasecomponent*
```

Comment:
hmmm, some doc-strings are `:raises Exception: description` while this one and others are `:raises: Exception <description>`. All of them need to be consistent but the linter doesn't seem to prefer one style over the other. 

Is there a difference between the two syntaxes ? If not I prefer to keep the current syntax in order to minimize the changes. If the new syntax is indeed what is needed then it should be changed in other places too but that doesn't seem related to the new linter.

Reply:
I remember the linter bringing up errors for some exceptions over this, will comb through again to find the offending ones.

Reply:
I've checked again and it seems like the linter only ever brings up errors if the Exception is explicitly raised in the method itself, unlike here where the `@permissions_required` would be the one to raise it or in other places where the model's manager querying the database would raise the exception. I guess the linter wouldn't be able to infer that there is a PerssionDenied exception being raised here.